bin/build_native.sh

```diff
@@ -2,25 +2,36 @@
 
```

Comment:
This script builds a container tagged `faasm/experiment-covid-native:VERSION` with the necessary dependencies and software to run `CovidSim`.